### DISCUSSION TITLE
Stript LED controller non comunica con il server

### DISCUSSION THREAD

**Discussion by davide1295** [indomus/forum#1342] [Networking]
Ciao a tutti, sto avendo uno strano problema con un controller per stript LED della Magic Home (ESP). Qualche giorno fa ho cambiato router (stessa marca, ASUS), riconfigurato tutti i dispositivi e fatto delle prove. Tutto funziona perfettamente (compresi altri controller simili ma di altre marche) tranne questo della Magic Home che nella pagina dei dispositivi connessi al router compare come connesso al wifi ma di fatto non funziona oppure esegue i comandi con un ritardo di qualche minuto. Dopo giorni di prove su prove sono giunto alla conclusione che se voglio farlo funzionare come si deve, il wifi deve essere impostato in legacy 20MHz. Vien da se che configurare il wifi in questo modo comporta ad avere pochissima banda disponibile (MAX 54Mbit) per gli altri dispositivi connessi (tra tutti una SMART TV). Il router è un ASUS e con quello che avevo prima, sempre ASUS, il controller funzionava benissimo se impostato in wifi n 40MHz.
Secondo voi quale potrebbe essere il problema?

**Comment by snakuzzo** [comment]
Sinceramente non saprei.
Io ho due magichome (esp826x) e li avevo inizialmente integrati con flux.led... erano inutilizzabili, c'erano dei ritardi imbarazzanti.
Li ho aperti e flashati con esphome. Ora vanno alla grandissima!

**Reply by davide1295** [reply]
Ritardi assurdi ne ho anche io oppure non funzionano proprio. A questo punto metterei anche io esphome, avresti qualche guida da passarmi al volo (sempre se si possa fare con il mio modello)?

**Reply by snakuzzo** [reply]
Prima cosa...apri la scocca e assicurati che sia un esp826x.
Una volta appurato questo, basta che cerchi su Google "magichome esphome" ;)

**Reply by davide1295** [reply]
Mi sono informato e non avendo HomeAssistant non ha molto senso mettere questo firmware.

**Reply by snakuzzo** [reply]
Ah ok...davo per scontato avessi un tuo hub personale
